backends/arm/operators/op_conv2d.py

```diff
@@ -59,12 +54,6 @@ def define_node(
         # Currently only int8 is supported in quantized types.
         actual_out_type = ts.DType.INT8 if is_quant_node else output.dtype
 
-        ## Transpose input tensor to NHWC_Order for TOSA
```

Comment:
Do we want to assert the dim order here?

Reply:
do you mean the `dim_order` of input and weight? 
but those orders are still in `NCHW` since they comes from the edge dialect. this fix patch is only assuming they're `NHWC` and is fixing those in the `arm_backend.py` for the output shape and op_placeholders for the input data and shapes.